lxd/storage/drivers/driver_powerflex_volumes.go

```diff
@@ -436,14 +436,15 @@ func (d *powerflex) commonVolumeRules() map[string]func(value string) error {
 		// ---
 		//  type: string
 		//  defaultdesc: same as `volume.block.type` or `thick`
-		//  shortdesc: Create a `thin` or `thick` provisioned volume
+		//  shortdesc: Whether to create a `thin` or `thick` provisioned volume
 		"block.type": validate.Optional(validate.IsOneOf("thin", "thick")),
 		// lxdmeta:generate(entities=storage-powerflex; group=volume-conf; key=size)
-		//
+		// The size must be in multiples of 8 GiB.
+                // See {ref}`storage-powerflex-limitations` for more information.
 		// ---
 		//  type: string
 		//  defaultdesc: same as `volume.size`
-		//  shortdesc: Size/quota of the storage volume in multiples of 8GiB
```

Comment:
Would it be wrong to have it there? It's a hard limit of PowerFlex. For example a user couldn't set a volume size of `6GiB` or `11GiB`. But I guess it's also mentioned in the general driver doc.

Reply:
Extra detail is generally supposed to be in the long description.

Reply:
Yep, it's still there once you expand the section. :)